cobc/replace.c

```diff
@@ -0,0 +1,313 @@
+/*
+   Copyright (C) 2003-2022 Free Software Foundation, Inc.
+   Written by Fabrice Le Fessant
```

Comment:
the date is fine, but the written by must include the original authors mentioned in the original file plus yours

Reply:
done